### PR TITLE
Step 7 part 1 & 2

### PR DESCRIPTION
**Pull request by noliver-ykc**
## What
(Optional) Task 1: Add a new item
(Optional) Task 2. Show item images

## CHECKS :warning:

https://github.com/noliver-ykc/mercari-build-training-2023/assets/74501096/30860be8-162a-48e1-8682-e1c2693ab15d

Please make sure you are aware of the following.

- [x] **The changes in this PR doesn't have private information

### CODE REVIEW

**Comment by noliver-ykc** [comment]
Please forgive my typo 人と商品を繋がる it should be 人と商品を繋げる, if i have time ill update it later!

File: python/main.py [hunk 1]
```diff
@@ -113,9 +119,10 @@ def read_items():
         INNER JOIN category ON items.category_id = category.id
     ''')
     item_results = cursor.fetchall()
+    items = [{"id": item[0], "name": item[1], "category": item[2], "image_filename": item[3]} for item in item_results]
```
Comment (karuna) [inline python/main.py:122]: is this correct?

Comment (noliver-ykc) [reply python/main.py:122]: hello! it does work as intended at this point, but if this code is incorrect in anyway (like we’re not supposed to this or something) please let me know and i’ll do a little more research!

Comment (rclarey) [reply python/main.py:122]: The logic looks good to me 👍

Comment (karuna) [reply python/main.py:122]: yeah, just confused with python syntax lol

File: python/images/default.jpg
Comment (iggyray) [inline python/images/default.jpg:1]: this is gold

**Review by iggyray — APPROVED** [review]
lgtm, Very cool!! Only issue is that I got (unjustly) rickrolled when reviewing code haha!

**Pull request merged**